spec/defines/resource_mailhost_spec.rb

```diff
@@ -48,31 +48,31 @@
           title: 'should enable IPv6',
           attr: 'ipv6_enable',
           value: true,
-          match: '  listen [::]:80 default ipv6only=on;'
+          match: '  listen                [::]:80 default ipv6only=on;'
         },
         {
           title: 'should not enable IPv6',
           attr: 'ipv6_enable',
           value: false,
-          notmatch: %r{  listen \[::\]:80 default ipv6only=on;}
+          notmatch: %r{  listen                \[::\]:80 default ipv6only=on;}
         },
```

Comment:
Is it possible to keep the spacing as it was before? In general, the module seems to use just a space or two after parameters, so I think it will be a little inconsistent if the spacing changes here. If we change the way we handle spacing, we'll probably need to do it on a module-wide basis.